mage_ai/cli/main.py

```diff
@@ -26,7 +27,17 @@ def list_commands(self, ctx: Context):
 
 INIT_PROJECT_PATH_DEFAULT = typer.Argument(..., help='path of the Mage project to be created.')
 INIT_PROJECT_TYPE_DEFAULT = typer.Option(
-    'standalone', help='type of project to create, options are main, sub, or standalone')
+    ProjectType.STANDALONE,
```

Comment:
I think you'll need to use `ProjectType.STANDALONE.value` here. Otherwise, the cli help message will show "ProjectType.STANDALONE" as the default value, which looks a bit weird.